CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Allow Ethstats connection url to specify ws:// or wss:// scheme. [#5494](https://github.com/hyperledger/besu/issues/5494)
 - Add support for Shanghai changes to the GraphQL service [#5496](https://github.com/hyperledger/besu/pull/5496)
 - Unite the tx-pool CLI options under the same Tx Pool Options group in UX. [#5466](https://github.com/hyperledger/besu/issues/5466)
+- Tidy DEBUG logs by moving engine_newPayload full logging to TRACE [#5529](https://github.com/hyperledger/besu/pull/5529)
```

Comment:
This change actually moves all engine_ API request logs to trace, which I think is OK but this needs updating